### PR TITLE
(#30) Add metadata to definition pages

### PR DESCRIPTION
**Pull request by sarinapadilla**
* Add fields needed for metadata setup to config
* Update configs for four dictionaries to include new fields
* Add metadata changes through helmet to definition rendering

* Added integration tests for meta tags
* Fixed LanguageToggle background

### CODE REVIEW

**Comment by bryanpizzillo** [comment]
The defaults for the react-app-dev testing selector need to be updated. I selected Cancer Terms (Spanish) and the hreflang links are incorrect. ES shows as the english url and en shows as the spanish URL.

File: cypress/integration/common/beforeEach.js [hunk 1]
```diff
@@ -9,12 +9,15 @@ beforeEach(() => {
     // to development defaults does not break a bunch of texts.
     win.INT_TEST_APP_PARAMS = { 
       audience: 'Patient',
-      basePath: '/',
```
Comment (seyilonge-nci) [inline cypress/integration/common/beforeEach.js:12]: Was `basePath` removed on purpose for `baseHost`?

Comment (sarinapadilla) [reply cypress/integration/common/beforeEach.js:12]: Copy-paste error! Updating to include baseHost and basePath.

Comment (seyilonge-nci) [reply cypress/integration/common/beforeEach.js:12]: 👍

**Pull request merged**